webapp/exercise_admin/templates/exercise_admin/file-upload-exercise-include-file-popup.html

```diff
@@ -5,17 +5,36 @@
 {% get_language_info_list for AVAILABLE_LANGUAGES as LANGUAGES_VERBOSE %}
 <div id="edit-included-file-{{ include_file.id }}" class="popup" data-file-id="{{ include_file.id }}">
   <div>
-    <h2 id="edit-included-file-title-{{ include_file.id }}" class="edit-included-file-title">{{ popup_title }}</h2>
-    {% if include_file.fileinfo.url  %}
-      <a href="{{ include_file.fileinfo.url }}" class="file-url">Uploaded file</a>
-    {% endif %}
+    <div class="edit-included-file-title-div">
+      {% for language_choice, _ in AVAILABLE_LANGUAGES %}{# TODO: Create a template tag filter for this #}
+        <span class="translated {% if language_choice == CURRENT_LANGUAGE and not create %}translated-visible{% endif %}" data-language-code="{{ language_choice }}">
```

Comment:
Instead of span use a div here. A span cannot contain, for example, heading tags.